smartsim/_core/mli/infrastructure/worker/worker.py

```diff
@@ -316,20 +332,31 @@ def __init__(self, result: bytes) -> None:
 class RequestBatch:
     """A batch of aggregated inference requests."""
 
-    requests: list[InferenceRequest]
-    """List of InferenceRequests in the batch"""
+    raw_model: t.Optional[Model]
+    """Raw bytes of the model"""
+    callbacks: t.List[CommChannelBase]
+    """The channels used for notification of inference completion"""
```

Comment:
I think we should switch these callbacks to descriptors (and create a ticket for doing the same in the InferenceReqeust). It's my fault, but I think this has caused some headaches with serialization... The worker should be constructing their own channels from the descriptor, instead.

Reply:
feel free to turn this into a follow up ticket. we want to avoid sending the actual channel around and just send descriptors so the sender can create what they need...

Reply:
Working on changing these to all be descriptors now!

Reply:
Finished this but will definitely need your opinions on it

Reply:
LGTM